modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -908,6 +918,16 @@ private void _addObjectDefinitionResources(
 								objectRelationship.getName());
 				}
 
+				Map<String, String> localeLabelMap =
+					objectRelationship.getLabel();
+
+				String objectRelationshipName = objectRelationship.getName();
+
+				localeLabelMap.computeIfAbsent(
+					defaultLocale, key -> objectRelationshipName);
```

Comment:
You can inline the `objectRelationshipName` just as you did with the `objectAction.getName()` @evibudai

Reply:
Nevermind, @marcelabc pointed out that this variable needs to be final/efectively final. So ignore my comment xD

Reply:
She can't do it here. It needs to be a final variable if I'm not mistaken.